curator-test/src/main/java/org/apache/curator/test/compatibility/CuratorTestBase.java

```diff
@@ -21,8 +21,11 @@
 import org.apache.curator.test.BaseClassForTests;
 import org.testng.annotations.Listeners;
 
-@Listeners(Zk35MethodInterceptor.class)
 public class CuratorTestBase extends BaseClassForTests
 {
+    public static final String zk35Group = "zk35";
+    public static final String zk36Group = "zk36";
+    public static final String zk35CompatibilityGroup = "zk35Compatibility";
```

Comment:
What's the difference between zk35 and zk35CompatibilityGroup?

Reply:
`zk35` is for tests that only work with ZooKeeper 3.5 or above (it's a pre-existing group). `zk35CompatibilityGroup` is a new group for tests that should be run by the `curator-test-zk35` module. I'll try to come up with better names.